src/main/java/org/jitsi/videobridge/simulcast/SimulcastSenderManager.java

```diff
@@ -134,8 +134,15 @@ public boolean accept(RawPacket pkt)
     private synchronized SimulcastSender getOrCreateSimulcastSender(
         SimulcastReceiver simulcastReceiver)
     {
-        if (simulcastReceiver == null
-                || !simulcastReceiver.isSimulcastSignaled())
+        if (simulcastReceiver == null)
+        {
+            return null;
+        }
+
+        SimulcastStream[] simulcastStreams
+            = simulcastReceiver.getSimulcastStreams();
+
+        if (simulcastStreams == null || simulcastStreams.length == 0)
```

Comment:
What?! Could you please write a comment why you're checking for simulcastStreams? I get it that it's necessary later on but that's under 2 other ifs there so I don't think it's obvious why you need to perform the check and return early and in all cases regardless of the later ifs.